scripts/run-pallet-benchmarks.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+read -p 'Pallet name: ' pallet
+
+pallet_name=$(echo "$pallet" | tr '-' '_')
+
+cargo run \
+--release \
```

Comment:
Generally see commands continued on the next line indented. ie.

```
cargo run \
  --release
```

Reply:
I was wondering about this